content/influxdb/cloud/reference/cli/influx/bucket-schema/create.md

```diff
@@ -0,0 +1,108 @@
+---
+title: influx bucket-schema create
+description: The `influx bucket-schema create` command creates a measurement schema for a bucket in InfluxDB.
+menu:
+  influxdb_cloud_ref:
+    name: influx bucket-schema create
+    parent: influx bucket-schema
+weight: 201
+related:
+  - /influxdb/cloud/organizations/bucket-schema
+---
+
+The `influx bucket-schema create` command creates a measurement schema for a bucket in InfluxDB.
+
+Creating a bucket schema requires a bucket of [schema-type **explicit**](influxdb/cloud/reference/cli/influx/bucket/create/#create-a-bucket-for-an-explicit-measurement-schema).
```

Comment:
```suggestion
{{% note %}}
To create a bucket schema, the bucket must have an [**explicit** schema-type](/influxdb/cloud/reference/cli/influx/bucket/create/#create-a-bucket-for-an-explicit-measurement-schema).
{{% /note %}}
```

Reply:
> To create..., the bucket....

doesn't quite work, which is what I was trying to avoid.